backend/src/main/java/com/woowacourse/levellog/authentication/config/LoginMemberResolver.java

```diff
@@ -2,7 +2,7 @@
 
 import com.woowacourse.levellog.authentication.support.AuthorizationExtractor;
 import com.woowacourse.levellog.authentication.support.JwtTokenProvider;
-import com.woowacourse.levellog.authentication.support.LoginMember;
+import com.woowacourse.levellog.authentication.support.Authentic;
```

Comment:
```suggestion
import com.woowacourse.levellog.authentication.support.Authentic;
import com.woowacourse.levellog.authentication.support.AuthorizationExtractor;
import com.woowacourse.levellog.authentication.support.JwtTokenProvider;
```
import 문 포맷팅 확인해주세요!